packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts

```diff
@@ -0,0 +1,174 @@
+import { sprinkles, vars } from '@kadena/react-ui/theme';
+
+import { $$borderColor } from '../../global.css';
+
+import { style, styleVariants } from '@vanilla-extract/css';
+
+export const treeListClass = style([
+  sprinkles({
+    padding: 0,
+  }),
+  {
+    listStyle: 'none',
+    overflowY: 'hidden',
+    transition: 'all .5s ease',
+  },
+]);
+
+export const treeListLevelVariantClass = styleVariants({
+  l1: {},
+  l2: {},
+  l3: {
+    marginLeft: vars.sizes.$1,
+    borderLeft: `1px solid ${$$borderColor}`,
+  },
+});
+
+export const treeListRootVariantClass = styleVariants({
+  isRoot: {
+    height: 'auto!important',
+  },
+  isNotRoot: {},
+});
+
+export const styledLinkClass = style([
+  sprinkles({
+    textDecoration: 'none',
+    color: '$foreground',
+    textTransform: 'capitalize',
+  }),
+  {
+    selectors: {
+      '&:hover': {
+        color: vars.colors.$primaryHighContrast,
+      },
+    },
+  },
+]);
+
+export const styledLinkActiveClass = styleVariants({
+  false: {},
+  true: { color: vars.colors.$primaryContrast },
+});
+
+export const levelItemVariantClass = styleVariants({
+  l1: {
+    display: 'block',
+    cursor: 'pointer',
+    fontWeight: vars.fontWeights.$bold,
+    padding: `${vars.sizes.$4} 0 ${vars.sizes.$2}`,
+    borderBottom: `1px solid ${$$borderColor}`,
+  },
+  l2: {
+    display: 'block',
+    cursor: 'pointer',
+    padding: `${vars.sizes.$2} 0`,
+    border: 0,
+  },
+  l3: {
+    display: 'block',
+    cursor: 'pointer',
+    padding: `0 ${vars.sizes.$3}`,
+    border: 0,
+  },
+});
+
+const markerStyle: Record<string, string> = {
+  content: '∙',
+  fontWeight: '$bold',
+  display: 'inline-block',
+  width: vars.sizes.$4,
+};
+
+export const levelItemLinkPseudoVariantClass = styleVariants({
+  l1: {},
+  l2: {
+    selectors: {
+      '&::before': markerStyle,
+    },
+  },
+  l3: {
+    selectors: {
+      '&::before': markerStyle,
+    },
+  },
+});
+
+export const treeItemPseudoMenuVariantClass = styleVariants({
+  'l1-true': {
+    selectors: {
+      '&::after': {
+        transform: 'translate(1px, 1px)  rotate(45deg)!important',
+      },
+    },
+  },
+  'l1-false': {},
+  'l2-true': {
+    selectors: {
+      '&::before': {
+        transform: `translate(${vars.sizes.$2}, ${vars.sizes.$1}) rotate(135deg)!important`,
```

Comment:
this variant needs to overwrite the other one.
and it seems that the order will not work. It needs an important